tests/system/src/test/java/test/robot/javafx/scene/NodeInitializationStressTest.java

```diff
@@ -0,0 +1,979 @@
+            c.getSelectionModel().select(0);
+            accessControl(c);
+        });
+    }
+
+    @Disabled("JDK-8349096") // FIX
+    @Test
+    public void menuButton() {
+        assumeFalse(SKIP_TEST);
+        test(() -> {
+            MenuButton c = new MenuButton();
+            c.setSkin(new MenuButtonSkin(c));
+            return c;
+        }, (c) -> {
+            c.getItems().setAll(new MenuItem("MenuButton"));
+            c.setPopupSide(Side.RIGHT);
+            accessControl(c);
+            c.show();
+        });
+    }
+
+    @Disabled("JDK-8349105") // FIX
+    @Test
+    public void pagination() {
+        assumeFalse(SKIP_TEST);
+        test(() -> {
+            Pagination c = new Pagination();
+            c.setSkin(new PaginationSkin(c));
+            return c;
+        }, (c) -> {
+            c.setPageFactory((pageIndex) -> {
+                return new Label(pageIndex + " " + nextString());
+            });
+            c.setPageCount(100);
+            c.setCurrentPageIndex(nextInt(100));
+            accessControl(c);
+        });
+    }
+
+    @Test
+    public void passwordField() {
+        assumeFalse(SKIP_TEST);
+        test(() -> {
+            PasswordField c = new PasswordField();
+            c.setSkin(new TextFieldSkin(c));
+            return c;
+        }, (c) -> {
+            accessTextInputControl(c);
+            c.setAlignment(Pos.CENTER);
+            c.getCharacters();
+        });
+    }
+
+    @Disabled("JDK-8349090") // FIX
+    @Test
+    public void pieChart() {
+        assumeFalse(SKIP_TEST);
+        test(() -> {
+            PieChart c = new PieChart();
+            c.getData().setAll(createPieSeries());
+            return c;
+        }, (c) -> {
+            c.getData().setAll(createPieSeries());
+            accessChart(c);
+        });
+    }
+
+    @Disabled("JDK-8347392") // FIX
+    @Test
+    public void radioButton() {
+        assumeFalse(SKIP_TEST);
+        test(() -> {
+            RadioButton c = new RadioButton("RadioButton");
+            c.setSkin(new RadioButtonSkin(c));
+            return c;
+        }, (c) -> {
+            accessControl(c);
+            c.setSelected(nextBoolean());
+        });
+    }
+
+    @Disabled("JDK-8349091") // FIX
+    @Test
+    public void scatterChart() {
+        assumeFalse(SKIP_TEST);
+        test(() -> {
+            ScatterChart c = new ScatterChart(createNumberAxis("x"), createNumberAxis("y"));
+            c.getData().setAll(createNumberSeries());
+            return c;
+        }, (c) -> {
+            c.getData().setAll(createNumberSeries());
+            accessChart(c);
+        });
+    }
+
+    @Test
+    public void scrollPane() {
+        assumeFalse(SKIP_TEST);
+        test(() -> {
+            ScrollPane c = new ScrollPane(new Label("ScrollPane"));
+            c.setSkin(new ScrollPaneSkin(c));
+            return c;
+        }, (c) -> {
+            c.setPannable(nextBoolean());
```

Comment:
Do you think it's worth adding some scrolling here? That might be a good follow-on test enhancement.

Reply:
follow-up